src-electron/generator/helper-zcl.js

```diff
@@ -887,6 +887,68 @@ async function zcl_atomics(options) {
   return templateUtil.templatePromise(this.global, promise)
 }
 
+/**
```

Comment:
Would adding similar logic to loader, and then setting a IS_BASELINE value for each atomic be more helpful in the future? Perhaps we can also suggest adding something in the XML that will help identify the baseline types.

Reply:
That is an alternative and being talked about in https://csamembers.slack.com/archives/C01B01GH64E/p1727461850620559

Reply:
This helper is not really needed so might actually remove it based on that conversation.

Reply:
Added that as well